### PR TITLE
Management command to load datasets

### PR DESCRIPTION
**Pull request by akmiller01**

### CODE REVIEW

**Comment by akmiller01** [comment]
@edwinmp I haven't been able to test the Box code yet, as I'm awaiting the App to be approved. But in theory it should work. Also, we need a way to get box_config.json to the server, which contains sensitive keys.

**Comment by akmiller01** [comment]
So I had to pull out the special requirements for this management command into `requirements_data.txt` Firstly, installing numpy and pandas on Travis was taking 21 minutes for tests to pass/fail, so I wanted to remove them from that. Second, they were causing spontaneous failures.

So once a deployment is made, if we want to reimport data we will need to
```
docker-compose exec web pip install -r requirements_data.txt
docker-compose exec web python3 manage.py datasectionexcel
```

File: di_website/datasection/mixins.py [hunk 1]
```diff
@@ -29,12 +30,16 @@ class Meta():
...
-            ('description', RichTextBlock(required=True)),
-            ('provenance', RichTextBlock()),
-            ('variables', RichTextBlock()),
-            ('geography', RichTextBlock()),
-            ('licence', RichTextBlock()),
-            ('citation', RichTextBlock())
+            ('description', TextBlock(required=True)),
+            ('provenance', TextBlock(required=False)),
+            ('variables', TextBlock(required=False)),
+            ('geography', TextBlock(required=False)),
```
Comment (edwinmp) [inline di_website/datasection/mixins.py]: Was the decision to make them `TextBlock`s for ease of import? Had made them RichText cos the pattern library seemed to have imaged they'd contain more than just text ... maybe some links at least... :)

File: di_website/datasection/models.py [hunk 2]
```diff
@@ -51,24 +51,33 @@ class DataSourceTopic(TaggedItemBase):
...
+    source_id = models.CharField(max_length=255, unique=True, blank=True, null=True)
+    title = models.TextField(unique=True)
     description = models.TextField(blank=True, null=True)
-    organisation = models.CharField(max_length=255, blank=True)
+    organisation = models.TextField(blank=True, null=True)
     link_to_metadata = models.URLField(blank=True)
-    geography = models.CharField(max_length=255, blank=True)
     link_to_data = models.URLField(blank=True)
+    geography = models.TextField(blank=True, null=True)
+    date_of_access = models.DateField(null=True, blank=True)
```
Comment (edwinmp) [inline di_website/datasection/models.py]: Duncan told me to remove this.

File: di_website/templates/datasection/includes/partials/dataset_body.html [hunk 3]
```diff
@@ -43,6 +43,27 @@
               </div>
             </div>
           {% endif %}
+          {% if page.figure_downloads.count %}
```
Comment (edwinmp) [inline di_website/templates/datasection/includes/partials/dataset_body.html]: Nice catch... 👍

**Pull request merged**